src/AspNet.Security.OAuth.Odnoklassniki/OdnoklassnikiAuthenticationOptions.cs

```diff
@@ -0,0 +1,51 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using static AspNet.Security.OAuth.Odnoklassniki.OdnoklassnikiAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.Odnoklassniki
+{
+    /// <summary>
+    /// Defines a set of options used by <see cref="OdnoklassnikiAuthenticationHandler"/>.
+    /// App registration could be performed by using next instruction: https://apiok.ru/dev/app/create.
+    /// After registration you will receive email with 3 parameters:
+    /// 1. Application ID - mapped to ClientId
+    /// 2. Public App key - mapped to PublicSecret
+    /// 3. Secret App key - mapped to ClientSecret
+    /// Email retrieval requires request to provider on api-support@ok.ru for GET_EMAIL permission (https://apiok.ru/ext/oauth/permissions).
+    /// </summary>
+    public class OdnoklassnikiAuthenticationOptions : OAuthOptions
+    {
+        public OdnoklassnikiAuthenticationOptions()
+        {
+            ClaimsIssuer = OdnoklassnikiAuthenticationDefaults.Issuer;
+
+            CallbackPath = OdnoklassnikiAuthenticationDefaults.CallbackPath;
+
+            AuthorizationEndpoint = OdnoklassnikiAuthenticationDefaults.AuthorizationEndpoint;
+            TokenEndpoint = OdnoklassnikiAuthenticationDefaults.TokenEndpoint;
+            UserInformationEndpoint = OdnoklassnikiAuthenticationDefaults.UserInformationEndpoint;
+
+            Scope.Add("GET_EMAIL");
+
+            ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "uid");
+            ClaimActions.MapJsonKey(ClaimTypes.Name, "name");
+            ClaimActions.MapJsonKey(ClaimTypes.Gender, "gender");
+            ClaimActions.MapJsonKey(ClaimTypes.Email, "email");
+            ClaimActions.MapJsonKey(ClaimTypes.Surname, "last_name");
+            ClaimActions.MapJsonKey(ClaimTypes.GivenName, "first_name");
+            ClaimActions.MapJsonKey(Claims.ImageUrl, "pic_2");
```

Comment:
What's the reasoning behind using `pic_2` instead of `pic_1` or `pic_3`?

Reply:
Just size. Is it better to map all pics than only one?

Reply:
I think it might just be a bit more consistent to support all 3, rather than just one arbitrarily.

Take a look at #241 and see what you think.